moonstone/parsers/counts/taxonomy/kraken2.py

```diff
@@ -5,7 +5,8 @@
 
 class SunbeamKraken2Parser(TaxonomyCountsBaseParser):
     """
-    Parse output from Kraken2 merge table from Sunbeam pipeline
+    Parse output from `Kraken2 <https://ccb.jhu.edu/software/kraken2/>`_
```

Comment:
Using the clearer way to link ('..' after two newlines) doesn't work with autosummary